c/top_level.mol

```diff
@@ -0,0 +1,8 @@
+import basic;
```

Comment:
Contract usually only needs `WitnessLayout`, I would recommend spliting the schema into two files:

- `witness_layout.mol`: Contains `WitnessLayout` and its dependencies
- `building_packet.mol`: This is for offchain, it imports `witness_layout` and includes the definition for `BuildingPacket` as well as other dependencies.